tests/acceptance/index-test.js

```diff
@@ -10,6 +12,23 @@ module('Acceptance | index', function(hooks) {
 
   hooks.beforeEach(function() {
     loadDefaultScenario(this.server);
+
+    this.clock = FakeTimers.install({
```

Comment:
The `issues` route template uses `{{moment-from-now}}` helper to display how much time has elapsed since an issue has been updated. We need to freeze time to get consistent Percy snapshots (and possibly, test assertions that check for the elapsed time).